client/i18n/locales/english/intro.json

```diff
@@ -2024,17 +2059,32 @@
       "fbbn": { "title": "179", "intro": [] },
       "lnmg": { "title": "180", "intro": [] },
       "wead": { "title": "181", "intro": [] },
-      "eekl": { "title": "182", "intro": [] },
+      "quiz-javascript-fundamentals": {
+        "title": "JavaScript Fundamentals Quiz",
+        "intro": [
+          "Test what you've learned in this quiz on JavaScript Fundamentals Quiz."
```

Comment:
this could be update to mention that is is a review of what you learned in the previous few modules. or something like that

Reply:
I dunno, these all could certainly be better - I just made a generic small description. But I think I would stick to describing it as a quiz - that's what it is. We have the review page for the reviews. It's pretty minor - we may want to revisit all these descriptions to come up with something better. I think anything is good enough for now.

Reply:
Makes sense 👍